### ISSUE TITLE
problem about reproduced results of LE+

### ISSUE THREAD

**Issue by zaiquanyang** [ExplainableML/czsl#8]
very admirable work!  But there is some confusing problems.
In the process of reproducing some methods such as Le+[1] based on your open source code, the results obtained are better than the results of Table 2 in article [2]. Is the published code somehow different from the setup in article [2]? Could you give some explanation?

[1] From redwine to red tomato: Composition with context
[2] Learning Graph Embeddings for Compositional Zero-shot Learning

**Comment by mancinimassimiliano** [comment]
Hi @zaiquanyang and sorry for the late reply.

The code, setup, and implementations are the same. However, results might vary across runs (slightly) and for each method we reported the model achieving the best AUC on the validation set. In case the differences are large, please let me and @ferjad know some more details on the results you got and the commands used to obtain them.

**Comment by ferjad** [comment]
@zaiquanyang [1] was using the old split, all papers after TMN (https://arxiv.org/pdf/1905.05908.pdf) use the new splits proposed by TMN, LE+ numbers were updated since, and our numbers are taken from previous papers. In practice, I found LE+ can do slightly better if you tune the hyperparameters more. Our implementation uses different lr and optimizers so the numbers are better though still maintaining the rankings of different methods.
